### ISSUE TITLE
Enable query parameter

### ISSUE THREAD

**Issue by Heiss** [jplatte/caniuse.rs#67]
Hey,

you created a great application. I would like to add it to my browser search bar with a shortterm. But for this, i need a query parameter.
In best case, it should looks like this: `https://caniuse.rs/?query=%s`.

Thank you and have a great day,
Heiss

**Comment by jplatte** [comment]
I agree this would be nice, but this site is quite low on my list of priorities these days. Contributions welcome!

**Comment by Heiss** [comment]
I would like to contribute. It helps a lot, if you could provide a workflow how i should implement it and point me to the starting point. I am a beginner in rust.

**Comment by jplatte** [comment]
I'm sorry, but the library (Yew) I used to create the site is something that I'm not that familiar with myself. You could try asking how to implement this sort of thing on their Discord or GitHub discussions.
